lib/entities/integrations/models.ts

```diff
@@ -0,0 +1,24 @@
+/** @format */
+
+import {IntegrationType} from "./enums";
+
+export type IIntegrationType = {
+    type: IntegrationType
+    value: string;
+};
+
+export interface IIntegrationData { }
+
+export interface IWebhooksIntegrationData extends IIntegrationData {
+    url: string;
+}
+
+/**
+ * Represents an Integration object.
+ */
+export type IIntegration = {
+    identifier: string;
```

Comment:
Is `identifier` at any point a string or is it a number? Just double checking here

Reply:
it is a string here (: